apis/coursebatchcertificateapi/coursebatchcertificateapi.yaml

```diff
@@ -0,0 +1,455 @@
+        $ref: '#/definitions/RemoveCourseBatchTemplateRequest'
+    required:
+    - request
+  RemoveCourseBatchTemplateRequest:
+    title: RemoveCourseBatchTemplateRequest
+    type: object
+    properties:
+      batch:
+        $ref: '#/definitions/RemoveCourseBatchTemplate'
+    required:
+    - batch
+  RemoveCourseBatchTemplate:
+    title: RemoveCourseBatchTemplate
+    type: object
+    properties:
+      courseId:
+        description: The course identifier
+        type: string
+      batchId:
+        description: The batch identifier
+        type: string
+      template:
+        $ref: '#/definitions/RemoveBatchTemplate'
+    required:
+    - courseId
+    - batchId
+    - template
+  RemoveBatchTemplate:
+    title: RemoveBatchTemplate
+    type: object
+    properties:
+      templateId:
+        description: The template identifier
+        type: string
+    required:
+    - templateId
+  IssueCertificateRequest:
+    title: Issue certificate request
+    type: object
+    properties:
+      request:
+        $ref: '#/definitions/Request'
+    required:
+    - request
+  Request:
+    title: Request
+    type: object
+    properties:
+      courseId:
+        description: The course identifier
+        type: string
+      batchId:
+        description: The batch identifier
+        type: string
+      userIds:
+        description: The optional User IDs
+        type: array
+        items:
+          type: string
+    required:
+    - courseId
+    - batchId
+  IssueCertificateResponse:
+    description: Issue certificate Response
+    type: object
+    properties:
+      result:
+        description: The response details
+        type: object
+        properties:
+          courseId:
+            description: The course identifier
+            type: string
+          batchId:
+            description: The batch identifier
+            type: string
+          status:
+            description: The certificate issue status
+            type: string
+  ResponseHeader:
+    title: ResponseHeader
+    type: object
+    properties:
+      id:
+        type: string
+        description: The API Identifier
+      ver:
+        type: string
+        description: The API version information
+      ts:
+        type: string
+        description: The API execution timestamp
+      params:
+        $ref: '#/definitions/ResponseParams'
+      responseCode:
+        type: string
+        description: 'The API response code '
+  ResponseParams:
+    title: Response Parameter
+    type: object
+    properties:
+      resmsgid:
+        type: string
+        description: Represents the response message ID
+      msgid:
+        type: string
+        description: Represents the error message ID
+      err:
+        type: string
+        description: Represents the error code
+      status:
+        type: string
+        description: The response status
+      errmsg:
+        type: string
+        description: Error Message
+  EmptyResult:
```

Comment:
@rahul-tarento @maheshkumargangula please clarify empty result description on lines 421 & 427

Reply:
In case of 400 bad request the 'result' fieldis an empty object hence the name EmptyResult